src/js/services/github-api-wrapper.ts

```diff
@@ -7,8 +7,27 @@ const GithubApiWrapper = async () => {
   const getReviewRequested = async (): Promise<PullRequest[]> => {
     const query = encodeURIComponent(`is:open is:pr review-requested:${userName} archived:false`);
     const pullRequests = await makeApiRequest('/search/issues', `q=${query}`);
+    const processedPullRequests = await processDataIntoPullRequests(pullRequests.items, false);
 
-    return processDataIntoPullRequests(pullRequests.items);
+    const teamPullRequestUrls = (await getTeamReviewRequested()).map(pr => pr.url);
+    return processedPullRequests.filter((pr) => !teamPullRequestUrls.includes(pr.url));
+  };
+
+  const getTeamReviewRequested = async (): Promise<PullRequest[]> => {
+    const teams = await SettingsStorageAccessor().loadTeams();
+    if (teams === '') return [];
+
+    let combinedPullRequests: Issue[] = [];
+
+    for (const team of teams.replace(/ /g, '').split(',')) {
+      const query = encodeURIComponent(`is:open is:pr team-review-requested:${team} archived:false`);
+      const pullRequests = await makeApiRequest('/search/issues', `q=${query}`);
+
+      /* istanbul ignore next */
```

Comment:
Not coverable by mock?

Reply:
It's quite hard to test... but I did it anyways now.